README.rst

```diff
@@ -210,6 +210,54 @@ It is not triggered when the device is created for the first time.
 Settings
 --------
 
+``OPENWISP_SSH_AUTH_TIMEOUT``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    | ``float`` |
```

Comment:
it can be float, but we're using integers, so I would write `int`.
Applies for the other settings as well.